### DISCUSSION TITLE
Influence of setting 'scaling' hyperparameter to 'no_scaling'

### DISCUSSION THREAD

**Discussion by Ruoyan0103** [ACEsuit/mace#1235] [General]
Hi,

I am currently investigating radiation damage using MACE. I observed that setting `no_scaling` improves the accuracy of predicted dimer energies at extremely short distances compared to not setting this parameter. My training dataset also includes forces. After reviewing previous discussions, it seems that `no_scaling` is typically not used. I would like to ask whether setting this parameter might have additional effects that I should be aware of.

Thanks.

**Comment by ilyes319** [comment]
Hey @Ruoyan0103, the only effect of no_scaling is to not multiply the forces and energies by a scalar number that usually helps the training dynamics. It is fine to not use scaling, especially if you have very big forces range like in your radiation damage.

Note that mace supports also ZBL pair repulsion which might help you for your research:

```
    --pair_repulsion \
    --distance_transform="Agnesi" \
```

**Reply by Ruoyan0103** [reply]
Many thanks! And actually I set this  
```
--pair_repulsion 
--distance_transform="Agnesi" 
```
in my config file. But only this these two hyperparameters cannot give me satisfying results. That's why I set 'no_scaling'.

**Reply by ilyes319** [reply]
that makes sense to me!
